config/service_providers.yml

```diff
@@ -387,3 +387,13 @@ production:
     cert: 'cbp_goes_pre_prod'
     redirect_uris:
       - 'http://10.156.152.27/login'
+
+  'urn:gov:dhs.cbp.jobs:openidconnect:aws-cbp-ttp':
+    agency: 'DHS'
+    allow_on_prod_chef_env: 'true'
+    block_encryption: 'aes256-cbc'
+    cert: 'cbp_goes_prod'
+    friendly_name: 'CBP Trusted Traveler Programs'
+    logo: 'cbp.png'
+    redirect_uris:
+      - 'https://ttp.cbp.dhs.gov'
```

Comment:
We use it to prefix-match callback URLs, and right now their app is deployed only inside of their network so the name doesn't resolve for us yet I think